docs/source/man.rst

```diff
@@ -336,16 +336,30 @@ Use the ``qpc scan add`` command to create scan objects with one or more sources
 
   Optional. The product inspection exclusion. Contains the list of products to exclude from inspection. Valid values are jboss_eap, jboss_fuse, and jboss_brms.
 
+``--enabled-extended-product-search=products_list``
+
+  Optional. The list of products to include for extended product search. Valid values are jboss_eap, jboss_fuse, and jboss_brms.
+
+``--ext-product-search-dirs=search_dirs_list``
```

Comment:
we should be consistent and either type out `extended` or use `ext` in both cases.  I'm a fan of `ext`; less typing.